OMOptim/build/OMOptim.pro

```diff
@@ -2,9 +2,7 @@ TEMPLATE = app
 TARGET = OMOptim
 
 QT +=  core gui svg xml
-greaterThan(QT_MAJOR_VERSION, 4) {
-    QT *= printsupport widgets webkitwidgets
-}
```

Comment:
More better if we made it a single line,

`QT += core gui svg xml printsupport widgets`